aioseop_class.php

```diff
@@ -4453,16 +4453,27 @@ function wp_head() {
 			echo "$meta_string\n";
 		}
 
-		// Handle Schema.
-		if ( version_compare( PHP_VERSION, '5.5', '>=' ) ) {
-			if ( ! empty( $aioseop_options['aiosp_schema_markup'] ) && boolval( $aioseop_options['aiosp_schema_markup'] ) ) {
-				$aioseop_schema = new AIOSEOP_Schema_Builder();
-				$aioseop_schema->display_json_ld_head_script();
-			}
-		} else {
-			if ( ! empty( $aioseop_options['aiosp_schema_markup'] ) && (bool) $aioseop_options['aiosp_schema_markup'] ) {
-				$aioseop_schema = new AIOSEOP_Schema_Builder();
-				$aioseop_schema->display_json_ld_head_script();
+		/**
+		 * The aioseop_disable_schema filter hook.
+		 *
+		 * Used to disable schema.org output programatically.
+		 *
+		 * @since 3.2.8
+		 *
+		 * @return boolean
+		 */
+		if ( apply_filters( 'aioseop_disable_schema', '__return_true' ) ) {
```

Comment:
Couple issues here.

1. Using `'__return_true'` remains a string, that option is for `add_filter()` and doesn't work the same for `apply_filter()`. When testing the hook, it was not a boolean, but instead was `'__return_true'`.  This would be better as a boolean (which seems to be the intent).
2. Would it be better to reverse the boolean so it makes more sense to other users as `add_filter( 'aioseop_disable_schema', '__return_true' );` rather than `add_filter( 'aioseop_disable_schema', '__return_false' );`?

I was thinking something like this would work. `if ( ! apply_filters( 'aioseop_disable_schema', false ) ) {`

Reply:
1) Good catch. I'll change that. 
2) Also a good catch. I had it backwards. :)